client/src/App.tsx

```diff
@@ -1,5 +1,7 @@
 import React from 'react';
-import { Typography } from '@material-ui/core';
+import { Typography, Button } from '@material-ui/core';
+
+import Validator from './Utils/Validations/Validator'
```

Comment:
why the import?

Reply:
Thats how you test the function

Reply:
maybe remove unused import? 
and import to the validator will be added when a phone input field is added
(also the Button import?)